scripts/explorer-jes-start.sh

```diff
@@ -30,8 +29,14 @@ EXPLORER_PLUGIN_NAME=$($NODE_BIN -e "process.stdout.write(require('./app/package
 # get current ui server directory
 SERVER_DIR="${ROOT_DIR}/components/explorer-jes/bin/server/"
 
+if [ -z "$ZOWE_PREFIX" ]
```

Comment:
So I'm not sure if the if block is needed, but we general have been using double square brackets in our if blocks. I think this should work in this case as you've quoted the variable name, but we've belt and braced most of them to cut out future surprises

Reply:
Removed the if condition, as it's not consistent with the other env vars